docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -37,11 +32,11 @@ To start using Dockerfiles the following conditions must be met:
 
 .. note::
 
-   OpenVINO's `Docker <https://docs.docker.com/>`__ and `Bare Metal <https://docs.openvino.ai/2023.0/ovms_docs_deploying_server.html#doxid-ovms-docs-deploying-server>`__ distributions are identical, so the documentation applies to both. 
+   OpenVINO's `Docker <https://docs.docker.com/>`__ and `Bare Metal <https://docs.openvino.ai/2023.0/ovms_docs_deploying_server.html#deploying-model-server-on-baremetal-without-container>`__ distributions are identical, so the documentation applies to both. 
```

Comment:
Since this is documentation about installing OpenVINO native tools, I don't understand why we're linking to OVMS documentation. IMO a link to the OV dev/runtime bare metal installations would be a better solution here: 

https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html

Reply:
The link here is for the user to get information about Bare Metal in OpenVINO, and since we have information about it in OVMS for now, that's why I put it there. If You think it is not necessary to have it in here I can remove it.

Reply:
In my opinion the link I provided above is a better reference to information about bare metal installation than the OVMS link for bare metal.

Reply:
Changed in https://github.com/openvinotoolkit/openvino/pull/19505/commits/996f027c571d8827f7d4a63fc5865681f00a398a